shared/data/constants.js

```diff
@@ -102,7 +114,8 @@ module.exports = {
         },
         {
             name: 'config',
-            url: 'https://staticcdn.duckduckgo.com/trackerblocking/config/v2/extension-config.json',
+            url: `https://staticcdn.duckduckgo.com/trackerblocking/config/v2/extension${browserName ? `-${browserName}` : ''}-config.json`,
```

Comment:
Please use a helper instead:

```suggestion
            url: `https://staticcdn.duckduckgo.com/trackerblocking/config/v2/${getConfigFileName()}`,
```


```
function getConfigFileName() {
    const browserInfo = parseUserAgentString()
    const suffix = browserName ? `-${browserName}` : ''
    return `extension${suffix}-config.json`
}
```

Reply:
Or optionally make the function return the entire path so no template string is needed:

```suggestion
            url: getConfigFileName(),
```